src/aics-image-viewer/components/shared/SmarterSlider.tsx

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import Nouislider, { NouisliderProps } from "nouislider-react";
+
+type CallbackArgs = [values: any[], handle: number, unencodedValues: number[], tap: boolean, positions: number[]];
+
+/** A wrapper around `Nouislider` that prevents updates while the slider is being dragged. */
+export default class SmarterSlider extends React.Component<NouisliderProps, { inactive: boolean }> {
+  constructor(props: NouisliderProps) {
+    super(props);
+    this.state = { inactive: true };
+  }
+
+  shouldComponentUpdate = () => this.state.inactive;
```

Comment:
consider renaming `inactive` to something more specific like `blockedUpdates` or even `shouldUpdate`